examples/stocks/lib/stock_home.dart

```diff
@@ -202,11 +201,7 @@ class StockHomeState extends State<StockHome> {
         Navigator.of(context).pushNamed('/stock/${stock.symbol}', mostValuableKeys: mostValuableKeys);
       },
       onShow: (Stock stock, Key arrowKey) {
-        showBottomSheet(
-          placeholderKey: _bottomSheetPlaceholderKey,
-          context: context,
-          child: new StockSymbolBottomSheet(stock: stock)
-        );
+        scaffoldKey.currentState.showBottomSheet(new StockSymbolBottomSheet(stock: stock));
```

Comment:
If you setState the widget that contains the scaffold, the bottom sheet won't rebuild.  It's the same issue we had with Drawer when showDrawer took a Widget instead of a WidgetBuilder.

You're not introducing this issue in this patch, but it's probably something we'll want to fix in a later patch.